tests/cases/fourslash/refactorConvertArrowFunctionOrFunctionExpression_Availability_Anon_FnParam.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference path='fourslash.ts' />

```

Comment:
- filename should be ...Argument.ts not Param
- filenames not consistent use ToAnon/ToArrow exclusively